<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QDialog" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="Window title">Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="1" colspan="3">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkOnTop">
         <property name="text">
          <string>Always on top</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkTray">
         <property name="text">
          <string extracomment="Infinitive">Hide to tray</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QComboBox" name="comboLang">
           <property name="maxVisibleItems">
            <number>15</number>
           </property>
           <item>
            <property name="text">
             <string extracomment="Means &quot;The system language&quot;">&lt;System&gt;</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="2" rowspan="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboNumberOfLists">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumContentsLength">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Language:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string extracomment="Means &quot;Number of lists:&quot;">Lists:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="checkRestoreIndustries">
         <property name="text">
          <string extracomment="Infinitive">Restore &quot;Industries&quot; window at startup</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="checkRestoreLP">
         <property name="text">
          <string extracomment="Infinitive">Restore link points at startup</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="checkSave">
         <property name="text">
          <string extracomment="Infinitive">Save position and size</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkSaveTickers">
         <property name="text">
          <string extracomment="Infinitive. Ticker is a short company name, see http://www.investopedia.com/terms/t/tickersymbol.asp . The common practice is to borrow &quot;ticker&quot; from English and incorporate into your language (see http://en.wikipedia.org/wiki/Loanword)">Automatically save tickers</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkAllowDups">
         <property name="text">
          <string extracomment="Infinitive. Ticker is a short company name, see http://www.investopedia.com/terms/t/tickersymbol.asp . The common practice is to borrow &quot;ticker&quot; from English and incorporate into your language (see http://en.wikipedia.org/wiki/Loanword)">Allow duplicates in a ticker list</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkMini">
         <property name="text">
          <string extracomment="Noun. Means &quot;The small window where the user can quickly enter a ticker name&quot;. Ticker is a short company name, see http://www.investopedia.com/terms/t/tickersymbol.asp . The common practice is to borrow &quot;ticker&quot; from English and incorporate into your language (see http://en.wikipedia.org/wiki/Loanword)">Fast ticker entry</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="checkHeader">
         <property name="text">
          <string extracomment="Infinitive">Show title</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="checkComments">
         <property name="text">
          <string extracomment="Infinitive. Means &quot;Automatically show comments to the selected ticker&quot;. Ticker is a short company name, see http://www.investopedia.com/terms/t/tickersymbol.asp . The common practice is to borrow &quot;ticker&quot; from English and incorporate into your language (see http://en.wikipedia.org/wiki/Loanword)">Automatically show ticker comments</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>54</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="checkButtons">
         <property name="text">
          <string extracomment="Infinitive">Show buttons</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkCtrlAltS">
         <property name="text">
          <string>Ctrl+Alt+S to take screenshots</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkCtrlAltR">
         <property name="text">
          <string>Ctrl+Alt+R to restore from tray</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>123</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="labelRestart">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string extracomment="Means &quot;It is neccessary to restart the application&quot;">Need restart</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="3">
    <widget class="QListWidget" name="list">
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="flow">
      <enum>QListView::TopToBottom</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>list</tabstop>
  <tabstop>comboLang</tabstop>
  <tabstop>comboNumberOfLists</tabstop>
  <tabstop>checkOnTop</tabstop>
  <tabstop>checkTray</tabstop>
  <tabstop>checkRestoreIndustries</tabstop>
  <tabstop>checkRestoreLP</tabstop>
  <tabstop>checkSave</tabstop>
  <tabstop>checkSaveTickers</tabstop>
  <tabstop>checkAllowDups</tabstop>
  <tabstop>checkMini</tabstop>
  <tabstop>checkHeader</tabstop>
  <tabstop>checkButtons</tabstop>
  <tabstop>checkComments</tabstop>
  <tabstop>checkCtrlAltS</tabstop>
  <tabstop>checkCtrlAltR</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Options</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Options</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboLang</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Options</receiver>
   <slot>slotLanguageChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotLanguageChanged(int)</slot>
 </slots>
</ui>
